assets/js/components/ClusterDetails/ClusterSettingsNew.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+
+import { getCluster } from '@state/selectors';
+import ChecksSelectionNew from './ChecksSelectionNew';
+
+export function ClusterSettingsNew() {
+  const { clusterID } = useParams();
+
+  const cluster = useSelector(getCluster(clusterID));
+
+  return <ChecksSelectionNew clusterId={clusterID} cluster={cluster} />;
+}
```

Comment:
Good job, we should move all the dispatches and the selectors here but I think it's something to be done in a second moment

Reply:
To be honest, I find this component not really useful, as it doesn't do anything. If we at least ported here all the dispatches, etc to make the testing of the check selection page more pure.
Or bring here the back button, the info box and all the "container" things.

What is the "functional" logic of this component, is we don't even have "settings" anymore?